File: app/src/main/java/com/young/simpledict/service/event/SearchWordRequest.java
package com.young.simpledict.service.event;

/**
 * Author: landerlyoung
 * Date:   2014-10-21
 * Time:   15:20
 * Life with passion. Code with creativity!
 */
public class SearchWordRequest extends BaseEvent {
   public String word;
   /**
    * use which dict, line {@link com.young.simpledict.dict.DictAdapter#DICT_YOUDAO_BRIEF}
    */
   public int useDict;
}

File: app/src/main/java/com/young/simpledict/service/event/SearchWordRequest.kt
package com.young.simpledict.service.event

/**
 * Author: landerlyang
 * Date:   2014-10-21
 * Time:   15:20
 * Life with passion. Code with creativity!
 */
class SearchWordRequest(eventCode: Any, var word: String, var dictToUse: Int) : BaseEvent(eventCode)
